### ISSUE TITLE
Overflowing player-title for very long album names

### ISSUE THREAD

**Issue by THeK3nger** [guregu/intertube#4]
The _King Gizzard & The Wizard Lizard_ broke _so many_ players with their long-ass album title :D On inter.tube, I've found this bad overflow. 

![CleanShot 2023-06-20 at 19 30 39@2x](https://github.com/guregu/intertube/assets/133159/c268ee39-78f2-42fb-befc-4ac101de07d1)

Playing with the devtools of the browser, I saw that adding  `margin-left: 100px` (more or less) to `.player-title` solves the problem.

(I can try to make a PR for this later, if it is okay for you.)

**Comment by guregu** [comment]
Oh man that is quite the song title. Thank you, PR would be appreciated! I will credit you :-)

**Comment by guregu** [comment]
I tried testing the fix but I can still break it with certain resolutions. I think we need to tweak it to use flexbox or something to hold the album art to a fixed size (e.g. 20vw) so we can offset the title by the same amount. I'll take another look once I get local dev mode finished.

**Comment by THeK3nger** [comment]
Yes! Definitely, a flexbox is the right approach. I also started experimenting with it, but without a local dev mode, I was not confident in the stability of what I did. :)

**Comment by guregu** [comment]
I just merged some stuff that makes it a lot easier to run locally, and also updated the readme. See if you can get it running!

**Comment by THeK3nger** [comment]
Great! I am running it locally without any problem. :D
